experiments/audio-player/index.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  // A helper exp-{name}-{var} class will be added to the root element
+  name: 'audio-player',
+
+  // Google optimize experiment id
+  experimentID: '8G1COSP2SWGSj8ZrLaERiw',
+
+  // [optional] specify number of sections for MVT experiments
+  // sections: 1,
+
+  // [optional] maxAge for a user to test this experiment
+  // maxAge: 60 * 60 * 24, // 24 hours,
+
+  // [optional] Enable/Set experiment on certain conditions
+  isEligible: ({ route }) => route.path === '/',
+
+  // Implemented variants and their weights
+  variants: [
+    { weight: 1 }, // Original
+    { weight: 1 } // Audio Player
```

Comment:
Are lines 19 & 20 supposed to be repeated?

Reply:
yes, one object for each experiment variation